<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Enter persons:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="typeHuman">
               <item>
                <property name="text">
                 <string>Select persons</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Doctor</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nurse</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Patient</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_ed_name"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Age:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_age"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_4">
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>301</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Select a person type to start entering data</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <widget class="QWidget" name="widget_2" native="true">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>371</width>
               <height>491</height>
              </rect>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>331</width>
                <height>161</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Work experience(year):</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spin_experian"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Number of successful operations:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spin_operations"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="page_3">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>371</width>
               <height>501</height>
              </rect>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>341</width>
                <height>131</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Qualification level:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="line_ed_qualif"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Number of procedures:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spin_procedures"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="page">
            <widget class="QWidget" name="widget" native="true">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>371</width>
               <height>471</height>
              </rect>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>10</y>
                <width>271</width>
                <height>61</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Diagnosis:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="diagnosis_line_edit"/>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QListWidget" name="list_inf"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_addperson">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_deleteperson">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
